components/Button/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ *
+ * Button.js
+ *
+ * A common button, if you pass it a prop "route" it'll render a link to a react-router route
+ * otherwise it'll render a link with an onclick
+ */
+
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Tooltip from 'antd/lib/tooltip';
+import StyledButton from './StyledButton';
+import A from './A';
```

Comment:
Please use meaningful names, if it's an anchor call it as such (i.e. `Anchor`), and also whether it's a wrapper, an anchor button or a link constructor?

Reply:
done - it's just a styled a tag.